components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -435,7 +436,8 @@ protected void concludeFlow(HttpServletRequest request, HttpServletResponse resp
             request.setAttribute("sessionDataKey", context.getCallerSessionKey());
             addAuthenticationResultToRequest(request, authenticationResult);
         } else {
-            FrameworkUtils.addAuthenticationResultToCache(context.getCallerSessionKey(), authenticationResult);
+            long timeoutPeriod = TimeUnit.MINUTES.toNanos(IdentityUtil.getOperationCleanUpTimeout());
```

Comment:
When expiry time is a property of the cache entry, why that needs to be passed. Shouldn't this be managed within the 'addAuthenticationResultToCache' method which adds the entry to the cache.